server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tservers/TabletServerInformation.java

```diff
@@ -32,11 +32,15 @@
 import org.apache.accumulo.monitor.rest.trace.RecoveryStatusInformation;
 import org.apache.accumulo.server.util.TableInfoUtil;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
 /**
  * Generates tserver information
  *
  * @since 2.0.0
  */
+@SuppressFBWarnings(value = "PA_PUBLIC_PRIMITIVE_ATTRIBUTE",
+    justification = "public member variables only referenced from tests, class used for serialization")
```

Comment:
Could also just add getters for each var and make them private but there are a lot of vars here so this suppression seems fine to me as well.